aspnetcore/security/authentication/disable-cookie/sample/Startup.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.UI;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using CookieAjax.Data;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CookieAjax
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContext<ApplicationDbContext>(options =>
+                options.UseInMemoryDatabase("app"));
+            #region snippet
+            services.AddAuthentication(o=>
```

Comment:
I think the example might be a little less involved without including Identity, using something like this:

```cs
services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
    .AddCookie(options =>
    {
        options.Events.OnRedirectToLogin = context =>
        {
            ...
        };
    });
```

If Identity *is* required here, it might be better to stick to using `AddIdentity` or event `AddDefaultIdentity` (rather than `AddIdentityCore`), and changing the above approach to using `services.ConfigurationApplicationCookie(...)`. I don't think I've seen `AddIdentityCore` or `AddIdentityCookies` described in the docs, so use of these here might be a little out of place. If that's something that makes sense, I can provide the relevant code.

Reply:
>  I can provide the relevant code.
Yes, that would help.

Reply:
I've created a [gist](https://gist.github.com/serpent5/1fb08fc1353124666629fbcae180028c) with an implementation of `ConfigureServices`.